tools/AutoTuner/README.md

```diff
@@ -0,0 +1,205 @@
+# AutoTuner with Ray cluster at GCP
+
+## Enable GKE
+
+Follow the Google quickstart guide up to the section "Create a GKE cluster"
+[here](https://cloud.google.com/kubernetes-engine/docs/quickstart). The
+quickstart guide will show how to enable GKE (Google's Kubernetes Engine)
+start a CLI interface and get the settings for your project.
+
+## Create a Kubernetes cluster
+
+Create GKE cluster using the following `gcloud` command as a guide.
+Note that each argument defines a characteristic of the cluster, further
+the cluster and node pool will allocate resources that cost money, so be
+mindful when choosing the configuration. For additional information see:
+
+```bash
+gcloud beta container clusters create --help
+gcloud beta container node-pools create --help
+```
+
+```bash
+gcloud beta container clusters create "autotuner" \
+         --machine-type "e2-standard-8" \
+         --image-type "UBUNTU_CONTAINERD" \
+         --disk-type "pd-standard" \
+         --disk-size "100" \
+         --num-nodes "1" \
+         --enable-autoscaling \
+         --min-nodes "1" \
+         --max-nodes "10"
+```
+
+Configure `gcloud` to use the Kubernetes credentials of the newly created
+cluster. The credentials allow the use of `kubectl` locally.
+
+```bash
+gcloud components install kubectl
+gcloud container clusters get-credentials autotuner
+```
+
+Create a new node pool. This step is optional, but recommended. The new pool
+will use preemptive nodes which are cheaper, but also has a more powerful
+CPU and have more disk space.
```

Comment:
Use present tense always (instead of will ...)
Change to ..
Create a new node pool.  (Demarcate the step in the process and then the description)
This step is optional, however, recommended. The new pool
uses preemptive nodes which is not only cheaper, but also has a more powerful
CPU and additional disk space.

Reply:
Fixed